packages/workspace/src/tasks-runner/run-command.ts

```diff
@@ -27,6 +37,14 @@ export async function runCommand<T extends RunArgs>(
     overrides
   );
 
+  projectsToRun.forEach((proj) => {
+    const root = proj.data.root;
+    loadEnvVars(`${root}/.local.env`);
+    loadEnvVars(`${root}/.env`);
+  });
+  loadEnvVars(`.local.env`);
+  loadEnvVars();
```

Comment:
@FrozenPandaz - is this a good place to load these? We have access to `proj.data.root`.

I discussed with Jack and it seems like another use-case that might be useful is loading in `.local.env` files - for devs that need a local, non-checked set of custom env values. But happy to discuss more and change this if we don't want to support this. 👍